akka-remote-tests/src/multi-jvm/scala/akka/remote/RemotingMultiNodeSpec.scala

```diff
@@ -48,4 +57,26 @@ abstract class RemotingMultiNodeSpec(config: MultiNodeConfig)
     }
     deleteFlightRecorderFile()
   }
+
+  /** Identify a user actor by actor name for the given `role`.
+   * {{{
+   *   runOn(first) {
+   *     val a = identify(second, "your-actor")
+   *   }
+   * }}}
+   */
+  def identify(role: RoleName, actorName: String, within: FiniteDuration = 10.seconds): ActorRef =
+    identifyWithPath(role, "user", actorName, within)
+
+  /** Identify an actor by `role / path / name`. Use for system actors as well. */
+  private[akka] def identifyWithPath(
+      role: RoleName,
+      path: String,
+      actorName: String,
+      within: FiniteDuration = 10.seconds): ActorRef = {
+    system.actorSelection(node(role) / path.split("/") / actorName).tell(Identify(actorName), probe.ref)
+    val actorIdentity = probe.expectMsgType[ActorIdentity](within)
+    assert(actorIdentity.ref.isDefined, s"Unable to Identify actor [$actorName] on node [$role]")
+    actorIdentity.ref.get
```

Comment:
Will migrate a few remote tests in a separate commit to use this where applicable. Same/similar function is duplicated in several tests.